platformio.ini

```
@@ -68,6 +68,14 @@ board = esp12e
 ; Comment out this line below if you have any trouble uploading the firmware
 ; and if it has a CP2102 on it (a square chip next to the usb port): change to 3000000 (3 million) for even faster upload speed
 upload_speed = 921600
+build_flags =
+  ${env.build_flags}
+  -D BOARD=BOARD_SLIMEVR_V1_2
+  -D VENDOR_NAME='"SlimeVR"'
+  -D VENDOR_URL='"https://slimevr.dev"'
+  -D PRODUCT_NAME='"SlimeVR Tracker v1.2"'
+  -D UPDATE_ADDRESS='"SlimeVR/SlimeVR-Tracker-ESP"'
+  -D UPDATE_NAME='"BOARD_SLIMEVR_V1_2-firmware"'
```

If someone is building from platformio.ini it is most likely a diy tracker.
At least for final we should change the values back to something that allows us to differentiate DIY and the normal Officials

```suggestion
build_flags =
  ${env.build_flags}
  -D VENDOR_NAME='"DIY"'
  -D VENDOR_URL='"https://docs.slimevr.dev"'
  -D PRODUCT_NAME='"DIY SlimeVR trackers"'
  -D UPDATE_ADDRESS='""'
  -D UPDATE_NAME='""'
```


Also the the following fields should be a described, how they have to be formated:
VENDOR_NAME, VENDOR_URL, PRODUCT_NAME, **UPDATE_ADDRESS, UPDATE_NAME**

But also do we need to set it here? Defaults would work too

---

I was not planning to keep it in, this piece got here by accident